docs/concepts/sides.md

```diff
@@ -34,10 +34,12 @@ How do we resolve this? Luckily, FML provides us with a `@SidedProxy` annotation
 
 !!! note
 
-    It is important to understand that FML picks the proxy to instantiate based on the physical side. A single player world (logical server, logical client within a physical client) will still have a proxy of the type you specify in `clientSide`! If you want behaviours that should run on both sides, make your client proxy extend from the server proxy and call the super class method. Make sure your server proxy does not reference any classes that only exist on the physical server.
+    It is important to understand that FML picks the proxy to instantiate based on the physical side. A single player world (logical server, logical client within a physical client) will still have a proxy of the type you specify in `clientSide`! If you want behaviours that should run on both physical sides, you must arrange for that to happen outside of the proxies
```

Comment:
I included it because people *have* been confused before as to why their "Server proxy" never runs at all in the physical client, because they think it's instantiated based on logical side (which makes no sense but people have still made that mistake)